apps/sync/test/unit/adapters/bookmarks_test.js

```diff
@@ -26,9 +26,8 @@ requireApp('sync/js/adapters/bookmarks.js');
 
 window.DataAdapters = {};
 
-suite('sync/adapters/bookmarks >', () => {
+suite('sync/adapters/bookmarks >', function() {
```

Comment:
Could you help to explain why to avoid using arrow-function? I suppose that is to keep the context inside the function the same with 'suite' provided. If so, what will the context be inside the function? Chai framework's 'this'?